Docs/Generali/piano_di_qualifica/04_test.tex

```diff
@@ -0,0 +1,92 @@
+    TS10 & Verifica che l'utente possa visualizzare le intestazioni della tabella & NI\tabularnewline
+    TS11 & Verifica che l'utente possa visualizzare i dati della tabella & NI\tabularnewline
+    TS12 & Verifica che l'utente possa visualizzare il dataset in 
+        forma di grafico 3D a barre verticali & NI\tabularnewline
+    TS13 & Verifica che l'utente possa visualizzare gli assi del grafico & NI\tabularnewline
+    TS14 & Verifica che l'utente possa visualizzare l'asse X con i valori appropriati & NI\tabularnewline
+    TS15 & Verifica che l'utente possa visualizzare l'asse Y con i valori appropriati & NI\tabularnewline
+    TS16 & Verifica che l'utente possa visualizzare l'asse Z con i valori appropriati & NI\tabularnewline
+    TS17 & Verifica che l'utente possa spostare la telecamera lungo una linea retta senza modificare l'angolazione della telecamera & NI\tabularnewline
+    TS18 & Verifica che l'utente possa spostare orizzontalmente la telecamera & NI\tabularnewline
+    TS19 & Verifica che l'utente possa spostare verticalmente la telecamera & NI\tabularnewline
+    TS20 & Verifica che l'utente possa compiere un'azione di zoom & NI\tabularnewline
+    TS21 & Verifica che l'utente possa ruotare il grafico  & NI\tabularnewline
+    TS22 & Verifica che l'utente possa ruotare il grafico attorno all'asse X & NI\tabularnewline
+    TS23 & Verifica che l'utente possa ruotare il grafico attorno all'asse Y & NI\tabularnewline
+    TS24 & Verifica che l'utente possa ruotare il grafico attorno all'asse Z & NI\tabularnewline
+    TS25 & Verifica che l'utente possa riposizionare la telecamera alla sua posizione iniziale & NI\tabularnewline
+    TS26 & Verifica che l'utente possa visualizzare i dettagli di una barra del grafico & NI\tabularnewline
+    TS27 & Verifica che l'utente possa visualizzare l'altezza della barra & NI\tabularnewline
+    TS28 & Verifica che l'utente possa visualizzare le etichette della barra & NI\tabularnewline
+    TS29 & Verifica che l'utente possa filtrare le barre che soddisfano certe condizioni & NI\tabularnewline
+    TS30 & Verifica che l'utente possa filtrare su un numero arbitrario (N) di barre & NI\tabularnewline
+    TS31 & Verifica che l'utente possa filtrare le N barre più alte & NI\tabularnewline
+    TS32 & Verifica che l'utente possa filtrare le N barre più basse & NI\tabularnewline
+    TS33 & Verifica che l'utente possa filtrare sul valor medio globale & NI\tabularnewline
+    TS34 & Verifica che l'utente possa filtrare le barre con altezza 
+        superiore al valor medio globale & NI\tabularnewline
+    TS35 & Verifica che l'utente possa filtrare le barre con altezza 
+        inferiore al valor medio globale & NI\tabularnewline
+    TS36 & Verifica che l'utente possa filtrare su una determinata barra & NI\tabularnewline
+    TS37 & Verifica che l'utente possa filtrare le barre con altezza 
+        superiore a una determinata barra & NI\tabularnewline
```

Comment:
ad una determinata

Reply:
La regola grammaticale
Si utilizza "A" quando la parola che segue inizia per consonante o per una vocale qualsiasi diversa da "a". (es. Andiamo a Otranto)

Si utilizza "AD" quando la parola che segue inizia per vocale "a". (es. Andiamo ad Ancona)

Vi sono anche delle eccezioni come "ad esempio" che si scrive in questo modo, anche se le vocali sono diverse, proprio perché suona meglio in questo modo.